src/shared/components/modlog.tsx

```diff
@@ -628,6 +684,94 @@ export class Modlog extends Component<any, ModlogState> {
     });
   }
 
+  setupUserFilter() {
+    if (isBrowser()) {
+      let selectId: any = document.getElementById("filter-user");
+      if (selectId) {
+        let config = choicesConfig;
+        config.classNames.containerOuter = "choices mb-2 custom-select col-4 px-0";
+        config.classNames.containerInner = config.classNames.containerInner + " py-0 default-font-size";
+        config.classNames.listSingle = config.classNames.listSingle + " py-0";
+        config.classNames.inputCloned = config.classNames.inputCloned + " w-100";
```

Comment:
These should be set in the choicesConfig in utils.ts , so they apply everywhere.

Reply:
These are specifically for the modlog dropdowns for filter by user/mod so that the style matches the filter by action dropdown (which is a different component).

Reply:
Make a single new choices config in utils.ts and use that for both of these. You have this single config copy-pasted twice.